ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndexer.scala

```diff
@@ -258,12 +262,42 @@ class JdbcIndexer private[index] (
             PersistenceEntry.Checkpoint(LedgerEntry.Checkpoint(recordTime.toInstant)))
           .map(_ => headRef = headRef + 1)(DEC)
 
-      case PartyAddedToParticipant(party, displayName, _, _) =>
+      case PartyAddedToParticipant(party, displayName, participantId, recordTime, submissionId) =>
         ledgerDao.storeParty(party, Some(displayName), externalOffset).map(_ => ())(DEC)
 
-      case PublicPackageUploaded(archive, sourceDescription, _, _) =>
+        //store party allocation accepted entry in party allocation entries table too
+        ledgerDao
+          .storePartyAllocationEntry(
```

Comment:
`storeParty` and `storePartyAllocationEntry` should be a single call, similar to how storing a `ledgerEntry` for a DAML transaction is done in the same database transaction as updating contracts, etc...

Reply:
`storeParty` is now gone and the `parties` and `party_allocation_entries` tables have been collapsed into `party_entries` so there is now just a single transaction.